static/js/components/Footer.js

```diff
@@ -8,7 +8,9 @@ class Footer extends React.Component {
          <div className="container container-footer">
             <div className="row">
                <div className="col-md-8">
-                  <img src="/static/images/mit-logo-ltgray-white@72x38.svg" alt="MIT" width="72" height="38" />
+                 <a href="http://www.mit.edu" target="_blank">
+                   <img src="/static/images/mit-logo-ltgray-white@72x38.svg" alt="MIT" width="72" height="38" />
+                 </a>
```

Comment:
Nitpick: is there a reason why the indentation doesn't match the way it used to?